### DISCUSSION TITLE
Zen Cart Community

### DISCUSSION THREAD

**Discussion by zcwilt** [zencart/zencart#5095] [General]
Currently the Community has three main focuses.

The [Zen Cart website](https://www.zen-cart.com), which is mainly focused on the forums and the downloads.

[Github](https://github.com/zencart/zencart), for those who want to get involved in development of Zen Cart code.

The [documentation project](https://docs.zen-cart.com/).

The Zen Cart website has not been updated in a long time, uses outdated vBulletin code and I admit is pretty ugly.

What can we do to make these more relevant to users and increase engagement.

**Comment by mc12345678** [comment]
If updating the search feature/capability is a difficult as it may seem, providing some guidance close to where searches are performed encouraging an internet search may be of benefit to users. E.g. a pop-up on hover that suggests to search via an internet tool.

**Comment by torvista** [comment]
Website: redesign to offer comprehensive information aimed totally at potential new users,  **ALL** on the landing page. Asking community to do mockups = more involvement. There are designers out there.

Forum: Big banner telling people to search the documentation first, before searching forum with Google. Keep an eye on resolution of issue and tell satisfied poster to open issue in documentation Github to include solution = more involvement.

Reporting code problems in Github: get people to search before adding an issue. As much as it will add a pile of non-issues to the project = more involvement. 

The more people that are pushed to Github, the more people realise there is active progress and that they can contribute.
